magnum_capi_helm/driver.py

```diff
@@ -736,6 +739,25 @@ def _update_helm_release(self, context, cluster, nodegroups=None):
             }
             values = helm.mergeconcat(values, network_details)
 
+        if self._get_k8s_keystone_auth_enabled(cluster):
+            k8s_keystone_auth_config = {
+                "authWebhook": "k8s-keystone-auth",
+                "openstack": {
+                    "k8sKeystoneAuth": {  # addon subchart configuration
+                        "enabled": True,
+                        "values": {
+                            "openstackAuthUrl": context.auth_url,
+                            "projectId": context.project_id,
```

Comment:
This might create issues with admin users updating clusters, but I don't want to block on that now, might be worth a follow up doing cluster.project_id instead here, for safety.